.travis.yml

```diff
@@ -1,7 +1,8 @@
 language: scala
 sudo: true
 scala:
-- 2.11.8
+- 2.11.11
```

Comment:
I cross-compiled `0.9.3`, so we need to use that. I just hit publish in sonatype, will need a while until its available.